layouts/partials/header.html

```diff
@@ -63,6 +63,7 @@
   {{ end }}
 
   {{ .Hugo.Generator }}
-  
+
+  {{ partial "custom_head.html" . }}
```

Comment:
Hey this is great thanks. Can I ask you to just comment it, something like `<!-- Custom head -->`?